ops/services/okta-global/main.tf

```diff
@@ -48,15 +54,37 @@ resource "okta_auth_server_claim" "given_name" {
   value          = "user.firstName"
 }
 
-resource "okta_auth_server_claim" "groups" {
+resource "okta_auth_server_claim" "simplereport_prod_roles" {
+  auth_server_id = data.okta_auth_server.default.id
+  claim_type = "RESOURCE"
+  name = "prod_roles"
+  value_type = "GROUPS"
+  group_filter_type = "STARTS_WITH"
+  value = "SR-PROD-"
+  scopes = [
+    okta_auth_server_scope.sr_prod.name]
+}
+
+resource "okta_auth_server_claim" "simplereport_stg_roles" {
   auth_server_id    = data.okta_auth_server.default.id
   claim_type        = "RESOURCE"
-  name              = "groups"
+  name              = "staging_roles"
   value_type        = "GROUPS"
-  group_filter_type = "REGEX"
-  value             = ".*"
-  scopes = [
-  okta_auth_server_scope.sr.name]
+  group_filter_type = "STARTS_WITH"
+  value             = "SR-STG-"
+  scopes            = [
+    okta_auth_server_scope.sr_stg.name]
+}
+
+resource "okta_auth_server_claim" "simplereport_devtest_roles" {
+  auth_server_id    = data.okta_auth_server.default.id
+  claim_type        = "RESOURCE"
+  name              = "devtest_roles"
+  value_type        = "GROUPS"
+  group_filter_type = "STARTS_WITH"
+  value             = "SR-TEST-"
+  scopes            = [
+    okta_auth_server_scope.sr_devtest.name]
 }
 
 // Create the CDC/USDS user groups
```

Comment:
The existing group ("Prime Team Members") doesn't correspond to any of the claims added in this PR. Is that group granted permissions somewhere outside of this file?

Reply:
Yes, that group has login permission (an application assignment) granted manually in the console. It has no specific permissions beyond that. ￼